### PR TITLE
Fix for blend_ics, remove boundary_long from xml

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
## DESCRIPTION OF CHANGES: 

- @hu5970 pointed out that the blend_ics task fails with the current version of the workflow.  This is because when I got rid of the get_extrn_ics task in PR #278, I got rid of the extrn_mdl_var_defns_fp file that was created during the get_extrn_ics task because it was no longer needed in make_ics.  I didn't realize blend_ics was also using this file.  EXTRN_MDL_CDATE appears to be the only variable that the blend_ics task needs from the extrn_mdl_var_defns_fp file, so I think the easiest solution is to create this file at the end of the make_ics task in extrn_mdl_staging_dir, which is now set to ${CYCLE_DIR}${SLASH_ENSMEM_SUBDIR}/ics/tmp_ICS in the make_ics task.  These changes have not been tested.  @hu5970 and @delippi please take a look and let me know what you think.
- The boundary_long cycle definition is removed from FV3LAM_wflow.xml since it is not needed for RRFS.  The make_lbcs_long task was just removed as part of PR #284 

### Machines/Platforms:

- WCOSS2
  - [ ] Cactus/Dogwood
  - [ ] Acorn
- RDHPCS
  - [ ] Hera
  - [ ] Jet
  - [ ] Orion
  - [ ] Hercules

### Test cases: 

- [ ] Engineering tests
  - [ ] Non-DA engineering test
  - [ ] DA engineering test
    - [ ] Retro
    - [ ] Ensemble
    - [ ] Parallel
- [ ] RRFS fire weather
- [ ] RRFS_A:
- [ ] RRFS_B:
- [ ] RTMA:
- [ ] Others:

### CODE REVIEW

**Review by MatthewPyle-NOAA — DISMISSED** [review]
Looks good to me, but will wait for confirmation from @delippi and/or @hu5970 before merging.

**Comment by BenjaminBlake-NOAA** [comment]
Thanks @MatthewPyle-NOAA, @hu5970 plans to test these changes so yes let's wait to hear from him before merging.

File: scripts/exrrfs_make_ics.sh
Comment (hu5970) [inline scripts/exrrfs_make_ics.sh:970]: 969 and 970 need to change to:
settings="EXTRN_MDL_CDATE=${extrn_mdl_cdate}"

Comment (BenjaminBlake-NOAA) [reply scripts/exrrfs_make_ics.sh:970]: Sure I will get that fixed now.  Having it on multiple lines was needed when there were multiple variables included in "settings".

**Comment by MatthewPyle-NOAA** [comment]
Is this one ready to go now?

**Comment by BenjaminBlake-NOAA** [comment]
I think so.  @hu5970 did this fix the date issue with blend_ics?

**Comment by ShunLiu-NOAA** [comment]
Please merge this PR. We are going to have another PR to use rrfs/rap dump instead of rrfs_e/rap_e dump.

**Review by MatthewPyle-NOAA — APPROVED** [review]
Approving and merging at the request of @hu5970

**Pull request merged**
